docs/_user_guide/julia-slurm.md

```diff
@@ -0,0 +1,59 @@
+# Running Julia on Yen-Slurm 
+
+## Single-threaded Julia on Yen-Slurm
+Similar to running Julia interactively, we will load Julia module but run the script using Slurm scheduler so that the script is executed on the Yen-Slurm nodes.
```

Comment:
The only time I used the hyphen, was referring to `yen-slurm`. I don't feel particularly strongly about the notation, so maybe we choose one and stick with it.